com.chartboost.helium/Android~/helium-android-bridge/src/main/kotlin/com/chartboost/heliumsdk/unity/HeliumUnityBridge.kt

```diff
@@ -0,0 +1,390 @@
+package com.chartboost.heliumsdk.unity
+
+import android.app.Activity
+import android.util.Log
+import com.chartboost.heliumsdk.HeliumIlrdObserver
+import com.chartboost.heliumsdk.HeliumImpressionData
+import com.chartboost.heliumsdk.HeliumSdk
+import com.chartboost.heliumsdk.ad.*
+import com.chartboost.heliumsdk.ad.HeliumBannerAd.HeliumBannerSize
+import com.chartboost.heliumsdk.unity.HeliumEventProcessor.HeliumBidEventConsumer
+import com.chartboost.heliumsdk.unity.HeliumEventProcessor.HeliumEventConsumer
+import com.chartboost.heliumsdk.unity.HeliumEventProcessor.HeliumRewardEventConsumer
+import com.chartboost.heliumsdk.unity.HeliumEventProcessor.serializeHeliumBidEvent
+import com.chartboost.heliumsdk.unity.HeliumEventProcessor.serializeHeliumEvent
+import com.chartboost.heliumsdk.unity.HeliumEventProcessor.serializeHeliumRewardEvent
+import com.chartboost.heliumsdk.unity.HeliumEventProcessor.serializePlacementILRDData
+import com.chartboost.heliumsdk.unity.HeliumUnityAdWrapper.Companion.wrap
+import com.unity3d.player.UnityPlayer
+
+@Suppress("NAME_SHADOWING")
+class HeliumUnityBridge {
+    private lateinit var _activity: Activity
```

Comment:
I would recommend making this `Activity?` as it's safer. In case people try to access the bridge before init; it won't crash. Might not do what they wanted, but at least it won't crash.

Reply:
This has been fixed